src/index.js

```diff
@@ -69,6 +69,22 @@ class IpfsRepo {
     await this.version.set(constants.repoVersion)
   }
 
+  /**
+   * Check if the repo is already initialized.
+   * @returns {Promise<Boolean>}
+   */
+  async isInitialized () {
+    try {
+      await this._openRoot()
+      await this._checkInitialized()
+      // necessary? await this.root.close()
```

Comment:
Is there a way to detect initialization without opening the root store?

Reply:
Only in node, in the browser you have to open the store first.